### ISSUE TITLE
Missing Player ID's in ff_scoringhistory()

### ISSUE THREAD

**Issue by TheMathNinja** [dynastyprocess/ffscrapr#286]
I'm running ffscrapr 1.4.2 but still having issues with this.

Running this code:

```
FAFL20 <- mfl_connect(season = 2020, league_id = 22686)
ff_scoringhistory(FAFL20, season = 2018:2020)
```
I'm getting NA mfl_id and sportradar_id for A.J. Brown in 2019 (but both are present for 2020).
Same for Demaryius Thomas in 2018 and 2019.

Is this just a completeness thing to be figured out later?

**Comment by tanho63** [comment]
Hmm. Missing sportradar on the nflfastr side I think, which breaks the join. 

You can probably get around it by joining dp_playerids via the gsis_id to get mfl_id, ideally this gets fixed nflfastr side though. I owe it a PR eventually.

**Comment by tanho63** [comment]
Resolved as of nflverse/nflfastr-roster#6
